### PR TITLE
refactor: simplify toggle control

### PR DESCRIPTION
**Pull request by kof**
## Description

Better interface, simpler

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Comment by TrySound** [comment]
Looks like flex wrap this is the only use case for toggle control which makes its behavior unexpected. It also does not give info which values user choose. Maybe switch to menu control like all other layout controls?

**Comment by kof** [comment]
I am switching menu control to the same interface as  toggle control in the next pr

File: apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx [hunk 1]
```diff
@@ -352,19 +352,12 @@ const LayoutSectionFlex = ({
...
-                  Icon: NoWrapIcon,
-                  value: "nowrap",
-                },
-              ]}
-              DefaultIcon={WrapIcon}
+              items={["wrap", "nowrap"]}
+              icons={{
+                wrap: WrapIcon,
+                nowrap: NoWrapIcon,
+                default: WrapIcon,
```
Comment (TrySound) [inline apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx:359]: what's the points of default if first item is default?

Comment (kof) [reply apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx:359]: well just to not have a logic inside the control that decides the first one is default?

Comment (kof) [reply apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx:359]: default is basically only ever needed when advanced value is used just to not have an empty one, maybe using the first would be fine?

Comment (kof) [reply apps/builder/app/builder/features/style-panel/sections/layout/layout.tsx:359]: actually lets kill the default

**Review by TrySound — APPROVED** [review]

**Pull request merged**